mephisto/scripts/local_db/gh_actions/auto_generate_requester.py

```diff
@@ -0,0 +1,33 @@
+from mdutils.mdutils import MdUtils
+from mephisto.client.cli_commands import get_wut_arguments
+from mephisto.operations.registry import (
+    get_valid_provider_types,
+)
+from mephisto.scripts.local_db.gh_actions.auto_generate_blueprint import (
+    create_blueprint_info,
+)
+
+
+def main():
+    requester_file = MdUtils(
+        file_name="../../../../docs/web/docs/reference/Requester.md",
+    )
+    requester_file.new_header(level=1, title="Requesters")
+    requester_file.new_paragraph("The requester is an account for a crowd provider.")
```

Comment:
@JackUrb is this accurate phraseology? Maybe we can align and borrow from the definitions in this doc: https://mephisto.ai/docs/explanations/architecture_overview/#crowdproviders

Reply:
Yeah, I got this from running `mephisto wut`. I need to update the statements in that command to match the docs.

Reply:
I think something like:
"`Requesters` are Mephisto's wrapper around an identity for a `CrowdProvider`, usually storing the credentials for an account to launch tasks from."
is probably in the right direction? But really anything you think gets this point across.